openml/runs/functions.py

```diff
@@ -161,6 +167,9 @@ def run_flow_on_task(
     upload_flow : bool (default=False)
         If True, upload the flow to OpenML if it does not exist yet.
         If False, do not upload the flow to OpenML.
+    dataset_format : str (default='dataframe')
+        If 'array', the dataset is read as a numpy array
```

Comment:
Could you please re-phrase as:
If 'array', the dataset is passed to the model as a numpy array.
(and the same for the dataframe, too).